doc/guide/wikiflow.md

```diff
@@ -0,0 +1,370 @@
+in [XML format](https://en.wikipedia.org/wiki/Help:Export) with the text of
+each page encoded in [Wiki Markup Language](https://en.wikipedia.org/wiki/Wikipedia:Wiki_Markup_Language).
+The `wikipedia-parsing` task takes these plus the Wikidata mapping and parses
+the documents into SLING document format. All the links in the Wikipedia
+articles are converted to Wikidata ids, and redirects are resolved to the target
+entity. The Wikipedia parser also outputs aliases from anchor text in the
+articles.
+
+Parsed [Wikipedia document](https://en.wikipedia.org/wiki/Annette_Stroyberg) for Wikidata item Q2534120:
+```
+Q2534120: {
+  =/wp/en/Annette_Stroyberg
+  :/wp/page
+  /wp/page/pageid: 488963
+  /wp/page/title: "Annette Stroyberg"
+  lang: /lang/en
+  /wp/page/text: "{{Use dmy dates|date=January 2014}}\n{{Infobox person\n|..."
+  /wp/page/item: Q2534120
+  :/s/document
+  /s/document/url: "http://en.wikipedia.org/wiki/Annette_Stroyberg"
+  /s/document/title: "Annette Stroyberg"
+  /s/document/text: "<b>Annette Strøyberg</b> (7 December 1936  – 12 December 2005) was a Danish actress..."
+  /s/document/tokens: [{=#1
+    :/s/token
+    /s/token/index: 0
+    /s/token/text: "Annette"
+    /s/token/start: 3
+    /s/token/length: 7
+  }, {=#2
+    :/s/token
+    /s/token/index: 1
+    /s/token/text: "Strøyberg"
+    /s/token/start: 11
+    /s/token/length: 10
+  }, {=#3
+    :/s/token
+    /s/token/index: 2
+    /s/token/text: "("
+    /s/token/start: 26
+    /s/token/length: 1
+  }
+  ...
+  ]
+  /s/document/mention: {=#401
+    :/wp/link
+    /s/phrase/begin: 13
+    name: "Danish"
+    /s/phrase/evokes: Q35
+  }
+  /s/document/mention: {=#402
+    :/wp/link
+    /s/phrase/begin: 14
+    name: "actress"
+    /s/phrase/evokes: Q33999
+  }
+  /s/document/mention: {=#403
+    :/wp/link
+    /s/phrase/begin: 19
+    /s/phrase/length: 3
+    name: "Les Liaisons Dangereuses"
+    /s/phrase/evokes: Q1498136
+  }
+  /s/document/mention: {=#404
+    :/wp/link
+    /s/phrase/begin: 34
+    /s/phrase/length: 2
+    name: "Roger Vadim"
+    /s/phrase/evokes: Q383420
+  }
+  ...
+  /wp/page/category: Q6135380
+  /wp/page/category: Q6547526
+  /wp/page/category: Q7482274
+  /wp/page/category: Q8362270
+}
+```
+
+# Name and phrase tables
+
+The aliases extracted from the parsed Wikipedia documents and Wikidata are
+consolidated to a set of aliases for each entity in the knowledge base in the
+`name-extraction` task. This alias table is used for producing a name table
+repository (`name-table` task) which contains all the (normalized) alias phrases
+in alphabetical order. This is useful for incremental entity name search used by
+the knowledge base browser.
+
+The `phrase-table` task creates a phrase table repository which can be used for
+fast retrieval of all entities having a (normalized) alias matching a phrase.
+From the Python SLING API you can use this for finding entities by name:
+```
+import sling
```

Comment:
I think we can remove the earlier section on PhraseTable since this is a more natural place. WDYT?

Reply:
I have removed the example, since this is a duplicate of the example in the pyapi doc. I have instead added a link to the phrase table section in pyapi doc.